ci/compare_sizes/project_instance.py

```diff
@@ -32,29 +32,20 @@ def _examples_dir(self):
         return os.path.join(self._root_dir, "examples")
 
     def _build_artifact(self, artifact, cache):
-        client = docker.from_env()
-        tag = "latest-arm64" if platform.machine() == "ARM64" else "latest-amd64"
-        image = f"nearprotocol/contract-builder:{tag}"
-
-        client.containers.run(
-            image,
-            "./build.sh",
-            mounts=[
-                docker.types.Mount("/host", self._root_dir, type="bind"),
-                docker.types.Mount(
-                    "/usr/local/cargo/registry", cache.registry, type="bind"
-                ),
-                docker.types.Mount("/usr/local/cargo/git", cache.git, type="bind"),
-                docker.types.Mount("/target", cache.target, type="bind"),
+        subprocess.run(
+            [
+                "cargo",
```

Comment:
this fails https://github.com/near/near-sdk-rs/actions/runs/14570961583/job/40868042720
try installing 0.14.0  `cargo-near` with a line similar to : 

https://github.com/near/cargo-near-new-project-template/blob/main/.github/workflows/deploy-production.yml#L17-L18

Reply:
this probably has to be done in a line, preceding https://github.com/near/near-sdk-rs/blob/master/.github/workflows/compare_sizes.yml#L32